packages/cdktf-cli/lib/cdktf-stack.ts

```diff
@@ -65,194 +69,56 @@ export type StackApprovalUpdate = {
   reject: () => void;
 };
 
+async function getTerraformClient(
+  abortSignal: AbortSignal,
+  stack: SynthesizedStack,
+  isSpeculative: boolean,
+  sendLog: (stateName: string) => (message: string, isError?: boolean) => void
```

Comment:
still called `sendLog` should be `createTerraformLogHandler` + `phase` here as well 🙈